documentation/modules/con-console.adoc

```diff
@@ -0,0 +1,12 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-using-console.adoc
+
+[id='con-console-{context}']
+= {ConsoleName} user permissions
+
+{ConsoleName} uses the OpenShift RBAC permissions model. For more information about the OpenShift RBAC permissions model, see the link:https://docs.openshift.com/container-platform/3.11/admin_guide/manage_rbac.html[OpenShift documentation^].
+
+To access the {ConsoleName}, the OpenShift user requires `addressspace` and `address` roles. The specific permissions given to the role govern the extent of the access. For example, for edit access, `edit` permissions need be to given to the role associated with the user, and for view-only access, `list` permissions need to be granted. For more information about the provided example roles, see link:{BookUrlBase}{BaseProductVersion}{BookNameUrl}#ref-example-roles-messaging[{ProductName} example roles].
```

Comment:
Terminology: Addressspaces and addresses are **resources**.  A **role** object defines a set of permissions that apply to one or more resources.

Reply:
Fixed

Reply:
I left one new suggestion.